src/wrappers/contract_wrappers/base_contract_wrapper.ts

```diff
@@ -4,6 +4,11 @@ import { BigNumber } from "../../../utils/bignumber";
 import * as Web3 from "web3";
 import * as singleLineString from "single-line-string";
 
+// Gas estimates can vary from the amount in practise. To prevent reverts,
+// we add a small amount to the estimate.
+const GAS_SAFETY_MARGIN = 2000;
+const MAXIMUM_GAS = 70000;
```

Comment:
I think this would make more sense as `MINIMUM_GAS`, given that it's functionally being used as a minimum.

Reply:
oh, urgh I actually meant Math.min in the function -- such that it won't exceed this number when adding the safety margin to the estimate.

Reply:
Does that make sense? Will update shortly.

Reply:
If that is the case and we're trying to make a maximum, it should probably be set to the block gas limit (which is around 4m)